src/constants.js

```diff
@@ -0,0 +1,28 @@
+export const emptyBook = {
```

Comment:
Постарайся такие статичиские данные писать с помощью uppercase
```suggestion
export const EMPTY_BOOK= {
```